docs/libreria-aserciones.md

```diff
@@ -0,0 +1,40 @@
+# Librerías de aserciones
+
+## Criterios de búsqueda de una librería de aserciones:
+
+Las librerías o bibliotecas de aserciones son un conjunto de funciones que nos permiten hacer ciertas validaciones para comprobar que nuestro código funciona correctamente. Puesto que en [#34](https://github.com/pablo1mc315/iv_pablomc/issues/34) hemos elegido MochaJS como test-runner y este no trae una librería de aserciones propia, debemos elegir una. 
+
+En cuanto a los criterios en los que me basaré para elegir una biblioteca de aserciones, voy a hacer uso de buenas prácticas y voy a comparar las que aparecen recomendadas en la [web oficial de MochaJS](https://mochajs.org/#assertions). Los criterios que deberá cumplir cada una para nuestro proyecto son:
+
+- Que sea una librería de aserciones que utilice BDD para que nos sea más fácil su uso y entendimiento.
+- Que esté actualizada y tenga un continuo mantenimiento, así como una comunidad bastante activa, para que nuestra aplicación no quede obsoleta en un futuro.
+
+## Librerías de aserciones que cumplen los criterios:
+
+### should.js
+
+- :heavy_check_mark: Es bastante conocido y tiene una comunidad mediana, aunque, por lo que he podido investigar, cada vez se está usando cada vez menos.
+- :x: El mantenimiento es su principal punto débil y lo cual lo descarta como una posible opción, puesto que no se actualiza desde marzo de 2019.
+
+### expect.js
+
+Es una librería de aserciones minimalista basada en should.js.
+
+- :heavy_check_mark: Tiene una comunidad bastante grande y está en continuo mantenimiento, por lo que eso no será un problema.
+
+### Chai
+
+Es una librería de aserciones que permite el tanto las prácticas BDD como TDD.
+
+- :heavy_check_mark: Tiene una comunidad muy grande y está en continuo mantenimiento, puesto que es el más utilizado, además de ser el que más puntuación tiene en comparadores como Snyk.
+- :heavy_check_mark: Como punto positivo, tiene mucha más documentación y tutoriales que cualquier otro, además de contar con su [propia web](https://www.chaijs.com/) en la que poder acceder a la documentación, plugings, etc., lo cual nos puede ayudar mucho a la hora de trabajar con él.
+
+### unexpected
```

Comment:
Sigue faltando una biblioteca, que además es estándar en node.js. Por favor, informaros un poco y no sigáis una sola fuente.

Reply:
Se ha modificado en el commit 0a66cd2e408c3a87fe32daf504fa5fa986aa6ff6. Corregiré lo que me ha revisado en cuanto a esta librería de aserciones en el comentario correspondiente y en un nuevo commit